From 3965c3bd4e45b10ca1f4b214ac0ce66221949bc1 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Mon, 25 Dec 2017 20:29:59 +0100
Subject: [PATCH] build: fix -version-info for libgf2x after ABI break

Commit e5c900c11e07d98c3074aea88a72e4107c2b3a8b changed the ABI and therefore
*must* change the -version-info numbers.

$ abidiff gf2x-1.1/.libs/libgf2x.so.1.0.0 gf2x-1.2/.libs/libgf2x.so.1.0.2
Functions changes summary: 16 Removed, 4 Changed, 20 Added functions
Variables changes summary: 0 Removed, 0 Changed, 1 Added variable

16 Removed functions:

  'function void gf2x_tfft_add(gf2x_tfft_info_srcptr, gf2x_tfft_ptr, gf2x_tfft_srcptr, gf2x_tfft_srcptr)'    {gf2x_tfft_add}
  [..]
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index b841724..0be4106 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -74,7 +74,7 @@ nobase_nodist_include_HEADERS=	\
 		gf2x/gf2x_mul9.h
 
 
-libgf2x_la_LDFLAGS=-version-info @gf2x_lib_version@ -no-undefined
+libgf2x_la_LDFLAGS=-release ${PACKAGE_VERSION} -no-undefined
 
 SUBDIRS=lowlevel src . fft tests $(MAYBE_APPS)
 
-- 
2.15.1

